web/blog/2022-05-31-filip-intro.md

```diff
@@ -0,0 +1,145 @@
+use it in my day-to-day.
+
+## What are you most excited about in Wasp?
+
+From a developer’s perspective, I’m definitely most excited about designing the
+DSL and working on the compiler.
+
+From a product standpoint, I love the idea of never again having to implement
+an authentication system.
+
+#### What’s a feature or project you’re most proud of that you worked on in the past three months?
+
+[Definitely this thing](https://www.youtube.com/watch?v=dQw4w9WgXcQ).
+
+#### How did you start coding?
+
+No exciting story to tell here, I’m afraid.
+
+I enrolled in computer science college and first started programming as a part
+of my regular “Introduction to programming” course.
+
+Before then, I didn’t really know what programming was. In fact, I  wanted to
+study electrical engineering. Unluckily for most but luckily for me, our
+college does not let you choose your major (i.e., electrical engineering or
+computing) right away. Instead, they make you go through a common year of
+universal (and some not-so-universal) courses, only allowing you to specialize
+at the start of your second year. Thanks to this quirk, I had a chance to
+encounter programming, change my mind, and enroll in computer science.
+
+#### Your dev setup?
+
+As far as hardware goes, it’s pretty basic. I use a regular laptop that’s
+starting to show signs of age (ASUS N550JX), a cheap Lenovo mouse, and my
+laptop’s integrated keyboard.
+
+The software setup is a bit more interesting. I’m usually on Manjaro Linux with
+the i3 tiling window manager. I jump between editors depending on what I’m
+doing. It’s usually either VS Code, Vim, or one of JetBrains IDEs, but I use
+Vim emulation no matter where I am. My preferred shell is zsh (sorry Frane).
+
+#### `camelCase` or `snake_case`?
+
+NoPreference
+
+#### What is your current favorite gem, library, tool, or anything else that helps you with your work? Why?
+
+That would have to be [ranger](https://github.com/ranger/ranger)!
+
+Ranger is a terminal file manager I use daily, mainly because it makes my work
+in the shell much faster. It also gives me a consistent interface for moving
+around, previewing files, and interacting with the file system. In other words,
+my workflow doesn't have to change much regardless of where I am (Mac, Linux
+Desktop, or Linux server).
+
+#### One piece of advice you’d give to budding developers?
+
+It’s easy to get stuck in an endless loop of polishing and fine-tuning your
+projects because you think they aren’t ready yet. Good enough sometimes truly
+is good enough and things can often be considered done before _you_ consider them
+done.
+
+I still occasionally need to give this advice to myself :)
+
+####  Lastly, where can people find or connect with you online?
+
+GitHub: [https://github.com/sodic](https://github.com/sodic)
+
+LinkedIn: [https://www.linkedin.com/in/filipsodic/](https://www.linkedin.com/in/filipsodic/)
+
+Email: filip@wasp-lang.dev
```

Comment:
Is banner below wrong since it has shayne's face? Which I love, but I thought you guys look a bit different.

Reply:
yep, I am now making a correct banner with Filip's face :D